LoRaEngine/deployment.lbs.template.json

```diff
@@ -107,6 +107,9 @@
                 },
                 "APPINSIGHTS_INSTRUMENTATIONKEY": {
                   "value": "$APPINSIGHTS_INSTRUMENTATIONKEY"
+                },
+                "REGION": {
```

Comment:
Without Region in the deployment file I was seeing errors in the LBS so I thought we should commit it to the template as well, any reason we shouldn't have it here @Mandur ?

Reply:
To be honest I think this was needed before @bastbu added the dynamic configuration. I think it is not needed anymore (at least CI is not populating it), could you please retry on latest dev?

Reply:
Verified and indeed the REGION config is not needed.